### DISCUSSION TITLE
可以添加杜比音效的解析

### DISCUSSION THREAD

**Discussion by diannaojiang** [the1812/Bilibili-Evolved#1412] [功能建议 / Ideas]
b站最新移动版客户端对部分视频上限了会限的杜比音效   希望脚本可以进行地址的解析
目前通过playurl进行获取可以在json里找到
`"dolby":{"audio":[],"type":"NONE"}}`
https://www.bilibili.com/bangumi/play/ep376298

**Comment by the1812** [comment]
但是 audio 里是空的...

**Reply by diannaojiang** [reply]
以目前的几个参数来获取确实是空的  但是在移动端上这个视频是有额外的杜比音效的  手头没有移动端tls抓包的工具
